src/components/LandingPage/Promo/GitStarPromoSection/GitStarPromoSection.module.css

```diff
@@ -47,15 +48,16 @@
 .button:focus {
   animation: pulse .8s;
   box-shadow: 0 0 2em 2em rgb(255 255 255 / 0%);
-  --hover: var(--bewebdev-pink-saturated);
+  --hover: gold;
```

Comment:
Use CSS variable with RGB/HSL instead of color preset (that can differ on different browsers).

Reply:
b023c18eeee44caa00c4a3d7c60b97efc81cea1d